src/roadmap/index.njk

```diff
@@ -93,7 +93,7 @@ title: Unison Roadmap
         <article class="roadmap_section">
           <header class="roadmap_section_header">
             <h2>Later</h2>
-            <p>Items that we are considering to work on in the future.</p>
+            <p>Things we are considering working on in the future.</p>
```

Comment:
clarifies that we are considering working on them, vs considering the things themselves

Reply:
this is one of those things where the more i look at it, the less good any option sounds